understanding/20/headings-and-labels.html

```diff
@@ -30,6 +30,9 @@ <h2>Intent of Headings and Labels</h2>
       <p>Labels and headings do not need to be lengthy. A word, or even a single character,
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
+       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive label without additional text. In these cases, authors should ensure that the image and its use as label in a particular context can be considered conventional and widely understood. 
+         Note that the same icon can be interpreted differently in different contexts and still be considered descriptive in that context. For example, placed after a search field, a magnifying glass icon is commonly interpreted as triggering a search. 
```

Comment:
Some suggested text for you to consider:
- swapped out the first instance of "icon" with "image", as previously "image" was the term being used, and prior to introducing the example, changing to "icon" seemed awkward.
- Split the first sentence into two, and carried over the "conventional and widely understood" caveat from the previous paragraph to explain why it can be still descriptive in different contexts.
- changed "when placed after" to "accompanying a" since the former was narrowly scoped and not representative of how all instances of search fields might be designed.
- similarly changed 'triggering' to the changed text because sometimes a search icon is merely a label, and sometimes it also acts as a button, but it always is a means to identifying the search widget.

```suggestion
         Note that the same image can be interpreted differently in different contexts. However, it can still be considered descriptive if its use is commonly understood in each context. For example, when accompanying a text field, a loupe or magnifying glass icon is commonly interpreted as indicating the field is for entering and submitting a search query. 
```